src/pages/VaccinePage.js

```diff
@@ -1,16 +1,104 @@
+/* eslint-disable react/forbid-prop-types */
 /**
  * mSupply Mobile
  * Sustainable Solutions (NZ) Ltd. 2016
  */
 
 import React from 'react';
-import { View } from 'react-native';
 import { connect } from 'react-redux';
+import { StyleSheet, Text, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
 
-export const VaccinePageComponent = () => <View />;
+import { FridgeDisplay, FlexView } from '../widgets';
 
-const mapStateToProps = state => state;
+import { selectMinAndMaxLogs, selectMinAndMaxDomains } from '../selectors/fridge';
+import { FridgeActions } from '../actions/FridgeActions';
 
-const mapDispatchToProps = dispatch => ({ dispatch });
+import { APP_FONT_FAMILY } from '../globalStyles';
+import { vaccineStrings } from '../localization';
+
+export const VaccinePageComponent = ({
+  selectedFridge,
+  minLine,
+  maxLine,
+  breaches,
+  maxDomain,
+  minDomain,
+  fridges,
+  onSelectFridge,
+}) => {
+  const Fridge = React.useCallback(
```

Comment:
Nitty nit: just for consistency, I've noticed you maybe prefer scoped `React` methods rather than destructuring them on import? I don't have an opinion either way, but I usually use `useCallback` over `React.useCallback`, so if there's a reason or just personal preference for using the `React` namespace I'll start doing it that way too!

Reply:
just easier.. less code to write 🤣 Very explicit and good for grokking since there are other `useX..` custom hooks n stuff .. personal perference no real reason. I'm just lazy